src/pages/understand-stacks/mining.md

```diff
@@ -62,3 +62,15 @@ To calculate the amount of BTC to send miners should:
 
 - Guess the price BTC/STX for the next day (100 blocks later)
 - Guess the total amount of BTCs committed by all miners
+
+## Microblocks
+
+Unlike most existing blockchains, Stacks blocks are not produced atomically. Instead, leaders may dynamically package transactions into a sequence of microblocks as they are received from users.
```

Comment:
This explanation is borrowed from the SIP. In this section, the idea was to explain what macroblocks are. Maybe it would be better to frame this as much:
- Anchor blocks "package" transactions and anchor them in a bitcoin block when mined
- Microblocks "stream" transactions to the miner for consideration for new anchor blocks